tests/model_explainability/evalhub/evalhub_kueue_integration/scripts/run_evalhub_tests.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Quick test runner for EvalHub Kueue integration tests
+
+export OC_BINARY_PATH="/opt/homebrew/bin/oc"
+export EVALHUB_BASE_URL="https://evalhub-prabhu.apps.rosa.prabhu-comhub.xqmp.p3.openshiftapps.com"
+export EVALHUB_MODEL_URL="http://granite-llm-metrics.prabhu.svc.cluster.local:8080/v1"
+export EVALHUB_NAMESPACE="prabhu"
+
+cd /Users/nbs/work/ai-engineering/code/opendatahub-tests
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)